dwave/cloud/client/base.py

```diff
@@ -530,6 +530,9 @@ def __init__(self, **kwargs):
         self._download_answer_executor = \
             ThreadPoolExecutor(self._DOWNLOAD_ANSWER_THREAD_COUNT)
 
+    class _Session(api.client.VersionedAPISessionMixin, BaseUrlSession):
```

Comment:
Why not use the `BaseUrlSessionMixin` here instead?

Reply:
For brevity. We could, but since `BaseUrlSession` is not deprecated, this is shorter.